app/src/main/java/com/sunasterisk/englishflashcard/data/model/Dictionary.kt

```diff
@@ -0,0 +1,10 @@
+package com.sunasterisk.englishflashcard.data.model
+
+data class Dictionary(
+    val id_dictionary: Int,
+    val word: String,
+    val word_type: String,
+    val spelling: String,
+    val translate: String,
+    val id_topics: Int
```

Comment:
```suggestion
    val id: Int,
    val word: String,
    val wordType: String,
    val spelling: String,
    val translate: String,
    val topicId: Int
```
Đặt tên theo camelCase, sửa lại cả những chỗ khác. Thêm endline nữa